src/todo.spec.js

```diff
@@ -215,13 +215,23 @@ describe('complete', () => {
     const stored = [{ id: 1, title: 'Todo 1', done: false }];
     const mockStore = createMockStore(stored);
 
-    const current = complete(mockStore, 1);
+    const current = completeTodo(mockStore, 1);
 
     expect(current).toStrictEqual({ id: 1, title: 'Todo 1', done: true });
     expect(mockStore.set.mock.calls[0][0]).toStrictEqual([
       { id: 1, title: 'Todo 1', done: true },
     ]);
   });
+
```

Comment:
it would be nice if you add more edge case like for example when params is a empty string.

However, it's a good move that you cleaned up app.js and moved the business logic into todo.js.